app/controllers/reports_controller.rb

```diff
@@ -100,6 +116,11 @@ def validate_collection_count_params
       { collection_ids: collection_ids, date_range: valid_dates }
     end
 
+    def validate_dpul_date_range
+      return nil if params[:date_range].blank? || !valid_dates
+      { date_range: valid_dates }
```

Comment:
When I run this in staging I get this error: https://app.honeybadger.io/projects/53391/faults/109834978.

When I run the code in the staging console, Plausible gives me this back:

```json
"{\"error\":\"Invalid format for `date` parameter. When using a custom period, please include two ISO-8601 formatted dates joined by a comma. See https://plausible.io/docs/stats-api#time-periods\"}"
```

I think the problem is that Plausible expects dates with no time component, we need to make sure we initialize this report with Dates, not DateTimes.